experiments/arctic_simulation.jl

```diff
@@ -63,7 +63,7 @@ using ClimaSeaIce.SeaIceMomentumEquations
 using ClimaSeaIce.Rheologies
 
 # Remember to pass the SSS as a bottom bc to the sea ice!
-SSS = view(ocean.model.tracers.S, :, :, grid.Nz)
+SSS = view(ocean.model.tracers.S.data, :, :, grid.Nz)
```

Comment:
does this stand for "sea surface salinity"?

Reply:
people use it... yeah